### ISSUE TITLE
Y axis label still sometimes overlaps ticks

### ISSUE THREAD

**Issue by openjck** [mozilla/metrics-graphics#838]
## Steps to reproduce

Load [this JSFiddle](https://jsfiddle.net/hhove4xz/2/)

## Actual result

![](https://user-images.githubusercontent.com/933396/37485702-9e7d890e-2862-11e8-80ec-3b6777dd0f67.png)

**Comment by cnwangjie** [comment]
May solve this problem by setting `left` argument a more large number manually.

**Comment by cnwangjie** [comment]
Following is the effect of setting `left: 80` in your example

![2018-03-29-211958_1920x1080_scrot](https://user-images.githubusercontent.com/7459652/38091139-313b1ede-3397-11e8-9025-dc867b758f79.png)

**Comment by openjck** [comment]
Good catch, @cnwangjie. I think it would be great if MG could automatically detect when this sort of thing is necessary and do it automatically.

**Comment by hamilton** [comment]
Yes. The tension we found was, many users expect the `left` to not automatically scale, especially if they're dealing with tons of graphs on a dashboard and are trying to get the design right. There doesn't really seem to be a paradigm that works 100% for everyone without some design tweaking or margin-nudging. I'd be open to changing this priority, but this was one thing we found when we were first making MG.

**Comment by hamilton** [comment]
The solution would probably be to do it automatically unless `left` is set. Usually if you're obsessively tweaking an MG graph (like I am) you're setting `left` anyway.
